src/test/java/ca/team2706/frc/robot/config/ConfigTest.java

```diff
@@ -114,6 +118,60 @@ public void setUp() throws Exception {
         }
     }
 
+
+    private static class FakeFluidConstant extends FluidConstant<String> {
```

Comment:
Should this be a delegate instead?

Reply:
I don't see how I would make that into a Delegate. Do you mind providing an example of what you mean?

Reply:
Actually, I think the only way that a delegate could be used would be to mock FluidConstant. What you have is fine.